main.nf

```diff
@@ -646,6 +647,67 @@ def d4_coverage_version(task) {
 	"""
 }
 
+process verifybamid2 {
+        cpus 16
+        memory '10 GB'
+        // publishDir "${OUTDIR}/contamination", mode: 'copy', overwrite: 'true', pattern: '*.selfSM'
+        tag "$id"
+        container = "${params.container_verifybamid2}"
+
+        input:
+                set group, id, file(bam), file(bai) from verifybamid2_bam
+
+        output:
+                file("${id}.result.selfSM")
+                file("${id}.result.Ancestry")
+                set group, file("*versions.yml") into ch_verifybamid2_versions
+
+        script:
+
+                if ( params.antype == "wgs") {
```

Comment:
This code could be made nicer by reusing the common part, and just having the `DisableSanityCheck` be inserted when running in non WGS mode.

But anyways, not a big deal, and now it is tested as is. So not a request for change, just a thought for future processes.

Reply:
I was just looking on how similar processes were built and took inspiration but if you think its redundant the way it is written I can change it?

Reply:
I think it is fine to leave as is here!